prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -117,10 +119,14 @@ def prompt_processor() -> Any:
         RunLevel.RUN,
         f"Preparing to execute {len(prompts)} prompt(s)",
     )
-
     # TODO: Rename "output" to "prompt"
     for output in prompts:  # type:ignore
         variable_names.append(output[PSKeys.NAME])
+        metadata[PSKeys.REQUIRED_FIELDS] = add_required_field(
+            metadata[PSKeys.REQUIRED_FIELDS],
+            output[PSKeys.NAME],
+            output.get(PSKeys.REQUIRED, None),
+        )
```

Comment:
Use by referance. please remove the return statement from the actual function.
```suggestion
        add_required_field(
            metadata[PSKeys.REQUIRED_FIELDS],
            output[PSKeys.NAME],
            output.get(PSKeys.REQUIRED, None),
        )
```

Reply:
done